osu.Game/Screens/Edit/Menus/EditorMenuBar.cs

```diff
@@ -32,9 +44,32 @@ private class DrawableEditorBarMenuItem : DrawableOsuMenuItem
             private Color4 openedForegroundColour;
             private Color4 openedBackgroundColour;
 
+            public override bool ReceiveMouseInputAt(Vector2 screenSpacePos) => parentSizedBox.ReceiveMouseInputAt(screenSpacePos);
+
+            /// <summary>
+            /// A box with width equal to this <see cref="DrawableEditorBarMenuItem"/>'s width, and height equal to the parent height.
+            /// </summary>
+            private readonly Box parentSizedBox;

```

Comment:
can you explain why this is a thing, not just what it is (pretty obvious what it is, but no idea why it has to exist)?

Reply:
Given the tabs are in the middle of that panel, the design shows:
1) The background of the items not being the full height of the panel
2) The submenu appearing just below the text (or just below the items).

On top of the above, I also want the whole height of the panel to result in input for that menu item - it felt more natural.

These present the following issues:
a) We can't make the menu items relative-Y, otherwise the submenu will appear at the bottom of the panel, much too far below the text.
b) We can't make the panels only autosized, since input on the panel will then not be handled.

So the way I countered this issue is by making them autosized (b) as per as the design indicates, but at the same time creating a hidden input receptor for the panel around them.

Reply:
as discussed, let's remove this and just use the actual size of the menu items (hover state needs to be adjusted with rounded grey box)